tests/test_data/test_loading.py

```diff
@@ -98,3 +100,99 @@ def test_load_seg(self):
         # this image is saved by PIL
         assert results['gt_semantic_seg'].shape == (288, 512)
         assert results['gt_semantic_seg'].dtype == np.uint8
+
+    def test_load_seg_custom_classes(self):
+
+        test_img = np.random.rand(10, 10)
+        test_gt = np.zeros_like(test_img)
+        test_gt[2:4, 2:4] = 1
+        test_gt[2:4, 6:8] = 2
+        test_gt[6:8, 2:4] = 3
+        test_gt[6:8, 6:8] = 4
+
+        img_path = osp.join(osp.dirname(__file__), 'img.jpg')
```

Comment:
I've tried using `tmpdir` and `tmp_path` for doing this, but it raises an error when calling `imwrite` (not sure why), so I've had to make this workaround.